inc/packages/templates/config.php

```diff
@@ -0,0 +1,222 @@
+	.es-package-status { float: right; text-transform: uppercase; padding: 2px 5px; font-size: 65%; border-radius: 3px; background: #152a4e; color: #fff; }
+	.es-package-status--active { background-color: #3FCF8E; color: #000; }
+	.es-package-status[class*="error"] { background-color: #ED7B9D; color: #152a4e; }
+</style>
+<div class="wrap">
+	<h1><?php esc_html_e( 'Search Configuration', 'altis' ); ?></h1>
+	<?php if ( $did_update ) : ?>
+		<?php if ( empty( $errors ) ) : ?>
+			<div class="notice notice-success is-dismissible">
+				<p><?php esc_html_e( 'Search configuration updated!', 'altis' ); ?></p>
+				<p><?php esc_html_e( 'Updates can take a few moments to propagate fully.', 'altis' ); ?></p>
+			</div>
+		<?php else : ?>
+			<?php foreach ( $errors as $error ) : ?>
+				<div class="notice notice-error is-dismissible">
+					<p><?php echo esc_html( $error->get_error_message() ); ?></p>
+				</div>
+			<?php endforeach; ?>
+		<?php endif; ?>
+	<?php endif; ?>
+	<p><?php esc_html_e( 'Configure search synonyms here to improve search results.', 'altis' ); ?></p>
+	<?php if ( is_network_admin() ) : ?>
+		<p class="description"><?php esc_html_e( 'These settings will be used as the default for all sites on the network that match the primary site language.', 'altis' ); ?></p>
+	<?php else : ?>
+		<p class="description"><?php esc_html_e( 'These settings will override the network wide settings if set for this site.', 'altis' ); ?></p>
+		<p class="description"><?php esc_html_e( 'Network wide settings are only applied if the primary language matches the language of this site.', 'altis' ); ?></p>
+	<?php endif; ?>
+	<hr />
+	<form action="" method="post" enctype="multipart/form-data">
+		<?php if ( get_locale() === 'ja' ) : ?>
+			<h2><?php esc_html_e( 'User Dictionary', 'altis' ); ?></h2>
+			<div class="columns">
+				<div class="column">
+					<p><?php esc_html_e( 'A user dictionary provides a way to control how words are broken up. If there are compound words or phrases specific to this site that users may search for they can be specified here to increase search relevancy.', 'altis' ); ?></p>
+					<p><?php esc_html_e( 'The syntax for the provided file should be in a CSV format:', 'altis' ); ?></p>
+					<pre><?php esc_html_e( 'text, token 1 ... token n, reading 1 ... reading n, part-of-speech tag', 'altis' ); ?></pre>
+					<p><?php esc_html_e( 'For example', 'altis' ); ?>:</p>
+					<pre>東京スカイツリー,東京 スカイツリー,トウキョウ スカイツリー,カスタム名詞</pre>
+				</div>
+				<div class="column">
+					<h3>
+						<label for="user-dictionary-file"><?php esc_html_e( 'File upload', 'altis' ); ?></label>
+						<?php if ( $user_dictionary_file_date && $user_dictionary_uploaded_status ) : ?>
+							<span class="es-package-status es-package-status--<?php echo esc_attr( strtolower( $user_dictionary_uploaded_status ) ); ?>">
+								<?php esc_html_e( 'Status', 'altis' ); ?>: <?php echo esc_html( $user_dictionary_uploaded_status ); ?>
```

Comment:
This needs to be done as a single string, not appended manually, for languages with varying orders.

```suggestion
								<?php echo esc_html( sprintf( __( 'Status: %s', 'altis' ), $user_dictionary_uploaded_status ) ) ?>
```